bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Display.java

```diff
@@ -117,6 +117,18 @@ enum APPEARANCE {
 	/* System property to be set for SWT application to use the system's theme */
 	static final String USE_SYSTEM_THEME = "org.eclipse.swt.display.useSystemTheme";
 
+	/**
+	 * This constant is used for configuring whether control.forceFocus() activates the parent shell.
+	 * Expects a <code>boolean</code> value.
+	 * To prevent control.forceFocus() to activate the shell, invoke
+	 * <pre>
+	 *   display.setData(Display.ACTIVATE_SHELL_ON_FORCE_FOCUS, false);
+	 * </pre>
+	 * after having created the display.
```

Comment:
Please provide information about expected **default** value / behavior.

Reply:
Done in the new pull request.